sdk/communication/Azure.Communication.PhoneNumbers/src/Models/PhoneNumbersModelFactory.cs

```diff
@@ -31,6 +33,8 @@ public static partial class PhoneNumbersModelFactory
         /// <param name="cost"> The incurred cost for a single phone number. </param>
 #pragma warning disable CA1054 // URI-like parameters should not be strings
         public static PurchasedPhoneNumber PurchasedPhoneNumber(string id, string phoneNumber, string countryCode, PhoneNumberType phoneNumberType, PhoneNumberCapabilities capabilities, PhoneNumberAssignmentType assignmentType, DateTimeOffset purchaseDate, PhoneNumberCost cost)
+#pragma warning restore CS0419 // Ambiguous reference in cref attribute
```

Comment:
Instead of disabling the warning we could remove the cref as in PhoneNumberSearchResult Initializes a new instance of PhoneNumberSearchResult